lib/menu.rb

```diff
@@ -0,0 +1,60 @@
+
+class Menu
+
+  attr_reader :basket, :receipt_list, :checkout
+
+  MEALS_LIST =
+  {
+    1 => ["Kimbab", 8],
+    2 => ["KFC", 6],
+    3 => ["Bibimbab_beef", 12.50],
+    4 => ["Bibimba_chicken", 10],
+    5 => ["Pork_bulgogi", 10],
+    6 => ["Kimchi", 4],
+    7 => ["Patato_salad", 5],
+    8 => ["Kimchi_pancake", 5],
+    9 => ["Rice", 3],
+    # 10 => ["Exit menu"],
+    11 => ["Checkout"]
+  }
+
+  def initialize(checkout_class = Checkout)
+    @checkout_class = checkout_class
+    @basket = []
+    @receipt_list = []
+  end
+
+  def print_menu
+    print MEALS_LIST
+  end
+
+  
+  def choose_meal(meal_number) # TODO : Has too many lines && NEEDS TESTING
```

Comment:
As you know there are too many lines, maybe using a switch statement for the various states of @meal_number could make it more readable